### PR TITLE
Update UI test for US2 US3 US5

### PR DESCRIPTION
**Pull request by vinotb-altmtrk**
Completed Update UI Test cases for US2(HW-55364), US3(HW-55365), US5(HW-55366)

### CODE REVIEW

File: transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java [hunk 1]
```diff
@@ -106,6 +127,9 @@ public void testSelectTransferMethod_verifyCorrectLabelsDisplayed() {
 
         mActivityTestRule.launchActivity(null);
 
+        verify(mHyperwalletInsight, atLeastOnce()).trackImpression(any(Context.class), anyString(), anyString(),
```
Comment (skoong) [inline transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java]: For all of these automated test cases we need to check that the function is called with the specific parameters related to the test.  For example in this one:
- pageName: transfer-method:add:select-transfer-method
- pageGroup: transfer-method
- Country: country of list shown
- Currency: currency of list shown

I'd suggest capturing the map to check that the country and currency are in the parameters

File: transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java [hunk 2]
```diff
@@ -446,4 +482,118 @@ public void testSelectTransferMethod_verifyDefaultsToUSWhenUserProfileDoesNotHav
...
+                .perform(RecyclerViewActions.actionOnItem(withChild(withText("PayPal Account")), click()));
+        onView(allOf(instanceOf(TextView.class), withParent(withId(R.id.toolbar)))).check(
+                matches(withText("PayPal Account")));
+
+        verify(mHyperwalletInsight, atLeastOnce()).trackClick(any(Context.class), anyString(), anyString(), anyString(),
+                ArgumentMatchers.<String, String>anyMap());
+    }
+
+    @Test
+    public void testSelectTransferMethod_verifyClickEventCreatedWhenUserMakesLongClickOnTransferMethod() {
```
Comment (skoong) [inline transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java]: You can remove this test, we don't need to automate this.

File: transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java [hunk 3]
```diff
@@ -446,4 +482,118 @@ public void testSelectTransferMethod_verifyDefaultsToUSWhenUserProfileDoesNotHav
...
+                .perform(RecyclerViewActions.actionOnItem(withChild(withText(R.string.bank_account)), longClick()));
+        onView(allOf(instanceOf(TextView.class), withParent(withId(R.id.toolbar)))).check(
+                matches(withText(R.string.title_add_bank_account)));
+
+        verify(mHyperwalletInsight, atLeastOnce()).trackClick(any(Context.class), anyString(), anyString(), anyString(),
+                ArgumentMatchers.<String, String>anyMap());
+    }
+
+    @Test
+    public void testSelectTransferMethod_verifyClickEventCreatedWhenUserMakesDoubleClickOnTransferMethod() {
```
Comment (skoong) [inline transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java]: You can remove this test as well, as it is not a critical.

File: transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java (hunk 1 above)
Comment (vinotb-altmtrk) [reply transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java]: Updated parameters.

File: transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java (hunk 3 above)
Comment (vinotb-altmtrk) [reply transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java]: Removed.

File: transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java (hunk 2 above)
Comment (vinotb-altmtrk) [reply transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java]: Removed.

**Comment by vinotb-altmtrk** [comment]
As discussed with @skoong I am closing this PR and creating new branch with updated branch #113

**Pull request closed without merging**